optaplanner-persistence/optaplanner-persistence-jaxb/pom.xml

```diff
@@ -63,8 +63,21 @@
       <artifactId>jaxb-impl</artifactId>
     </dependency>
     <dependency>
-      <groupId>javax.activation</groupId>
-      <artifactId>activation</artifactId>
+      <groupId>jakarta.activation</groupId>
+      <artifactId>jakarta.activation-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.json</groupId>
+      <artifactId>jakarta.json-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>jakarta.json</artifactId>
+    </dependency>
+    <!-- MOXy needed for JSON -->
+    <dependency>
+      <groupId>org.eclipse.persistence</groupId>
+      <artifactId>org.eclipse.persistence.moxy</artifactId>
```

Comment:
I do not understand the need for this. We're a JEE company, we have our own JSON/JAXB stuff. :-) 
See RESTEasy Jackson 2 Provider, which should also come with Quarkus by default.

Reply:
The purpose of this PR is to add JSON marshalling support for JAXB. Jackson already gives us XML and JSON support, but we're trying to give users the choice of using JAXB as well.

Reply:
RESTEasy can be used for that as well, [see the docs](https://docs.jboss.org/resteasy/docs/4.4.2.Final/userguide/html_single/index.html#JAXB_+_JSON_provider). I know, because I use it for the exact same purpose in my own projects.

Either that, or I don't understand how your use case differs. I've been wrong before.

Reply:
From my understanding, this `optaplanner-persistence-jaxb` module is responsible for implementing JSON marshalling using the JAXB standard, so I'm looking for a JAXB provider (not Jackson, a module for that already exists in `optaplanner-persistence-jackson`) that supports JSON marshalling (MOXy is the provider used in this PR).

RESTEasy is a JAX-RS implementation. It does support JSON marshalling, but it does so by using its own JAXB provider. Since this module is for JAXB only, I don't think we should introduce RESTEasy dependencies into it. We just need a JAXB provider.

After some thought, I think I approached this the wrong way. @triceo I see what you're saying now, I could just use Quarkus' RESTEasy module to do this in the OptaPlanner extension, rather than adding MOXy to handle JAXB JSON.

Reply:
Yes, for the extension, I definitely think that reusing what is already there in Quarkus is the way to go. 

FYI, plain Jackson can be used with JAXB as well, without the use of RESTEasy:
https://github.com/FasterXML/jackson-modules-base/tree/master/jaxb

Reply:
Yup, that does it for Quarkus.

Hmm, this JAXB Jackson dependency looks interesting, it seems like an alternative to adding JAXB annotated JSON support in `optaplanner-spring-boot-autoconfigure`. Not sure if it belongs in `optaplanner-persistence-jaxb`, since we already have a module for Jackson. @ge0ffrey thoughts on these alternatives?